core/templates/css/oppia.css

```diff
@@ -159,26 +159,26 @@ why they need to be overriden.  */
   transform: translate3d(0,-1px,0);
 }
 
-.oppia-learner-confirm-button.protractor-test-submit-answer-button.md-button[disabled],
-.oppia-learner-confirm-button.protractor-test-submit-answer-button.mat-button[disabled] {
+.oppia-learner-confirm-button.e2e-test-submit-answer-button.md-button[disabled],
```

Comment:
Unrelated to this PR, is it ok to use e2e test classes for styling? +cc @kevintab95?

Reply:
Oh shoot, no, we shouldn't be using e2e related classes for styling. @ShivamJhaa can you please file an issue for this? Thanks!

Reply:
Done. Issue #15654